microsite/data/plugins/scaleops.yaml

```diff
@@ -0,0 +1,9 @@
+title: ScaleOps
+author: terasky.com
+authorUrl: https://github.com/terasky-oss
+category: Kubernetes
+description: View Rightsizing and resource usage data for your Kubernetes workloads right from within Backstage!
+documentation: https://github.com/TeraSky-OSS/backstage-plugins/tree/main/plugins/scaleops-frontend
```

Comment:
The [documentation here](https://github.com/TeraSky-OSS/backstage-plugins/tree/main/plugins/scaleops-frontend#configuration) is a little worrisome. When will you have a proper secured version ready? Maybe it would be best to wait for that to be in place.

Reply:
the docs were outdated and support was already added for scaleops auth but not for all of the auth methods. i updated the docs with supported auth methods. currently i dont plan on adding additional auth methods proactively and am waiting for community feedback and requests to add them in, as we do not utilize the other methods in our environments.

Reply:
This part in the proxy config is still a concern: `credentials: dangerously-allow-unauthenticated`. Do the other options now work? Ideally the default for this should be `require` that way you need at minimum a Backstage token to call this endpoint.

Reply:
ive checked and that is no longer needed. i removed that setting. thanks for the pointer. indeed this was there initially but is not needed anymore so good to get rid of it